Runtime/Scripts/SelectionTools/PolygonInput.cs

```diff
@@ -382,6 +406,12 @@ private void CloseLoop(bool isNewPolygon, bool checkPreviewLine = true)
 
         private void Tap()
         {
+            if (mode == DrawMode.Edit)
+            {
+                Debug.LogWarning("PolygonInput is in edit mode, cannot Add a new point in edit mode.", gameObject);
```

Comment:
Ik heb deze warnings er bewust in gelaten voor mogelijke debugging bij het gebruik van de package